icons/gitkraken/gitkraken-plain-wordmark.svg

```diff
@@ -0,0 +1 @@
3 0 0 0-.195 1.04 2.338 2.338 0 0 0 1.17 2.075 45.266 45.266 0 0 1-5.262 81.768v-18.9a7.274 7.274 0 0 0 5.066-6.883 7.144 7.144 0 0 0-3.572-6.3c1.688-16.302 9.156-12.015 9.156-17.21V33.94C91.244 26.08 73.19.752 64.748.103h-1.56c-8.442.65-26.496 25.978-26.496 33.836v2.988c0 5.195 7.468.908 9.156 17.21a7.274 7.274 0 0 0 1.558 13.183v18.9a45.266 45.266 0 0 1-5.324-81.768 2.338 2.338 0 0 0 .973-3.115 2.273 2.273 0 0 0-2.143-1.3 2.143 2.143 0 0 0-1.103.261 50.007 50.007 0 0 0 12.34 92.35V71.347a39.097 39.097 0 0 0 4.74 1.232v25.395A54.424 54.424 0 0 1 9.609 44.07a55.073 55.073 0 0 1 3.442-19.16 2.338 2.338 0 0 0-1.233-2.987 2.598 2.598 0 0 0-.976-.195 2.403 2.403 0 0 0-2.207 1.558 59.1 59.1 0 0 0 52.93 79.817V73.23h4.806v29.875a59.035 59.035 0 0 0 56.697-59.035 57.866 57.866 0 0 0-3.767-20.784 2.338 2.338 0 0 0-4.547.846 1.624 1.624 0 0 0 .13.78 53.644 53.644 0 0 1 3.442 19.158 54.424 54.424 0 0 1-47.215 53.904V72.579a39.097 39.097 0 0 0 4.74-1.234V92.65A50.007 50.007 0 0 0 88.194.298a2.403 2.403 0 0 0-1.242-.297Zm-8.043 55.485a4.87 4.87 0 0 1 2.854 8.392 4.87 4.87 0 1 1-6.883-6.885 4.87 4.87 0 0 1 4.03-1.507zm-29.261.09a4.806 4.806 0 0 1 3.41 1.417 4.87 4.87 0 0 1 0 6.885 4.806 4.806 0 0 1-6.819 0 4.87 4.87 0 0 1 0-6.885 4.806 4.806 0 0 1 3.409-1.417zm-36.012 53.568c-5.325 0-9.611 3.896-9.611 9.351a9.352 9.352 0 0 0 9.548 9.418c3.247 0 7.144-1.622 8.832-5.39a11.3 11.3 0 0 0 .78-4.612H12.207v2.989h6.819a5.196 5.196 0 0 1-1.883 2.728 6.235 6.235 0 0 1-3.637 1.104c-2.857 0-5.846-2.145-5.846-6.237a5.91 5.91 0 0 1 5.846-6.17 5.26 5.26 0 0 1 3.506 1.235 6.04 6.04 0 0 1 1.365 1.494h4.092c-1.17-3.312-4.742-5.91-8.834-5.91zm12.21.326v2.856h3.182v-2.856zm7.468 0v4.676H31.04v2.467h2.272v10.976h3.183v-10.976h2.338v-2.467h-2.338v-4.676zm8.119 0v18.12h3.7v-8.12l6.042 8.12h4.416l-7.209-9.095 6.56-9.025H51.11l-5.977 8.377v-8.377zm41.435 0v18.12h3.182v-6.69l4.871 6.69h3.637l-5.586-7.534 5.586-5.91h-3.832l-4.676 5.129v-9.805zm-18.77 4.285a3.832 3.832 0 0 0-3.702 2.143v-1.752h-3.051v13.443h3.182v-7.273c0-2.273 1.168-3.38 3.44-3.38h.132zm7.534 0c-3.767 0-6.82 3.248-6.82 7.08s2.533 7.143 6.82 7.143a5.066 5.066 0 0 0 4.547-2.338v1.95h3.182v-13.444h-3.182v1.947a5.066 5.066 0 0 0-4.547-2.338zm30.07 0a6.82 6.82 0 0 0-4.937 2.21 7.079 7.079 0 0 0 .39 10.066 7.014 7.014 0 0 0 10.002-.26 6.69 6.69 0 0 0 1.819-3.637 7.728 7.728 0 0 0-.586-4.287l-2.272 2.404a3.507 3.507 0 0 1-.91 3.57 3.767 3.767 0 0 1-5.13.585l7.468-7.922a7.404 7.404 0 0 0-5.844-2.729zm17.08 0c-2.403-.065-3.57 1.105-4.156 2.21v-1.82h-2.988v13.444h3.183v-7.404c0-1.884.778-3.442 3.051-3.442s2.924 1.559 2.924 3.573v7.273h3.182v-8.119c0-2.338-.78-5.715-5.196-5.715zm-92.935.39v13.444h3.181v-13.443zm75.576 2.39a3.637 3.637 0 0 1 2.162.597l-5.26 5.522a3.572 3.572 0 0 1 .455-4.807 3.637 3.637 0 0 1 2.643-1.312zm-29.336.208a3.897 3.897 0 0 1 3.701 2.338 3.767 3.767 0 0 1 .39 1.883 4.806 4.806 0 0 1-.519 1.883 3.702 3.702 0 0 1-3.572 2.145 4.027 4.027 0 0 1-4.092-4.092 4.027 4.027 0 0 1 4.092-4.157z" class="cls-1"/></svg>
```

Comment:
Found just two small things
- Add newline at EOF
- remove  `class="cls-1"` since it has no effect here (?)

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#179287" d="M86.951.001a2.403 2.403 0 0 0-2.07 1.336 2.923 2.923 0 0 0-.195 1.04 2.338 2.338 0 0 0 1.17 2.075 45.266 45.266 0 0 1-5.262 81.768v-18.9a7.274 7.274 0 0 0 5.066-6.883 7.144 7.144 0 0 0-3.572-6.3c1.688-16.302 9.156-12.015 9.156-17.21V33.94C91.244 26.08 73.19.752 64.748.103h-1.56c-8.442.65-26.496 25.978-26.496 33.836v2.988c0 5.195 7.468.908 9.156 17.21a7.274 7.274 0 0 0 1.558 13.183v18.9a45.266 45.266 0 0 1-5.324-81.768 2.338 2.338 0 0 0 .973-3.115 2.273 2.273 0 0 0-2.143-1.3 2.143 2.143 0 0 0-1.103.261 50.007 50.007 0 0 0 12.34 92.35V71.347a39.097 39.097 0 0 0 4.74 1.232v25.395A54.424 54.424 0 0 1 9.609 44.07a55.073 55.073 0 0 1 3.442-19.16 2.338 2.338 0 0 0-1.233-2.987 2.598 2.598 0 0 0-.976-.195 2.403 2.403 0 0 0-2.207 1.558 59.1 59.1 0 0 0 52.93 79.817V73.23h4.806v29.875a59.035 59.035 0 0 0 56.697-59.035 57.866 57.866 0 0 0-3.767-20.784 2.338 2.338 0 0 0-4.547.846 1.624 1.624 0 0 0 .13.78 53.644 53.644 0 0 1 3.442 19.158 54.424 54.424 0 0 1-47.215 53.904V72.579a39.097 39.097 0 0 0 4.74-1.234V92.65A50.007 50.007 0 0 0 88.194.298a2.403 2.403 0 0 0-1.242-.297Zm-8.043 55.485a4.87 4.87 0 0 1 2.854 8.392 4.87 4.87 0 1 1-6.883-6.885 4.87 4.87 0 0 1 4.03-1.507zm-29.261.09a4.806 4.806 0 0 1 3.41 1.417 4.87 4.87 0 0 1 0 6.885 4.806 4.806 0 0 1-6.819 0 4.87 4.87 0 0 1 0-6.885 4.806 4.806 0 0 1 3.409-1.417zm-36.012 53.568c-5.325 0-9.611 3.896-9.611 9.351a9.352 9.352 0 0 0 9.548 9.418c3.247 0 7.144-1.622 8.832-5.39a11.3 11.3 0 0 0 .78-4.612H12.207v2.989h6.819a5.196 5.196 0 0 1-1.883 2.728 6.235 6.235 0 0 1-3.637 1.104c-2.857 0-5.846-2.145-5.846-6.237a5.91 5.91 0 0 1 5.846-6.17 5.26 5.26 0 0 1 3.506 1.235 6.04 6.04 0 0 1 1.365 1.494h4.092c-1.17-3.312-4.742-5.91-8.834-5.91zm12.21.326v2.856h3.182v-2.856zm7.468 0v4.676H31.04v2.467h2.272v10.976h3.183v-10.976h2.338v-2.467h-2.338v-4.676zm8.119 0v18.12h3.7v-8.12l6.042 8.12h4.416l-7.209-9.095 6.56-9.025H51.11l-5.977 8.377v-8.377zm41.435 0v18.12h3.182v-6.69l4.871 6.69h3.637l-5.586-7.534 5.586-5.91h-3.832l-4.676 5.129v-9.805zm-18.77 4.285a3.832 3.832 0 0 0-3.702 2.143v-1.752h-3.051v13.443h3.182v-7.273c0-2.273 1.168-3.38 3.44-3.38h.132zm7.534 0c-3.767 0-6.82 3.248-6.82 7.08s2.533 7.143 6.82 7.143a5.066 5.066 0 0 0 4.547-2.338v1.95h3.182v-13.444h-3.182v1.947a5.066 5.066 0 0 0-4.547-2.338zm30.07 0a6.82 6.82 0 0 0-4.937 2.21 7.079 7.079 0 0 0 .39 10.066 7.014 7.014 0 0 0 10.002-.26 6.69 6.69 0 0 0 1.819-3.637 7.728 7.728 0 0 0-.586-4.287l-2.272 2.404a3.507 3.507 0 0 1-.91 3.57 3.767 3.767 0 0 1-5.13.585l7.468-7.922a7.404 7.404 0 0 0-5.844-2.729zm17.08 0c-2.403-.065-3.57 1.105-4.156 2.21v-1.82h-2.988v13.444h3.183v-7.404c0-1.884.778-3.442 3.051-3.442s2.924 1.559 2.924 3.573v7.273h3.182v-8.119c0-2.338-.78-5.715-5.196-5.715zm-92.935.39v13.444h3.181v-13.443zm75.576 2.39a3.637 3.637 0 0 1 2.162.597l-5.26 5.522a3.572 3.572 0 0 1 .455-4.807 3.637 3.637 0 0 1 2.643-1.312zm-29.336.208a3.897 3.897 0 0 1 3.701 2.338 3.767 3.767 0 0 1 .39 1.883 4.806 4.806 0 0 1-.519 1.883 3.702 3.702 0 0 1-3.572 2.145 4.027 4.027 0 0 1-4.092-4.092 4.027 4.027 0 0 1 4.092-4.157z" /></svg>

```